src/main/java/net/dv8tion/jda/api/entities/Activity.java

```diff
@@ -47,13 +48,15 @@
      *
      * @return RichPresence or {@code null} if {@link #isRich()} returns {@code false}
      */
+    @Nullable
     RichPresence asRichPresence();
 
     /**
      * The displayed name of the {@link Activity Activity}. If no name has been set, this returns null.
      *
-     * @return Possibly-null String containing the Activity's name.
+     * @return String containing the Activity's name.
      */
+    @Nonnull
```

Comment:
This was originally `Possibly-null` but the `Notnull` annotation was add. Is that correct?

Reply:
The `EntityBuilder` uses `String.valueOf(json.get("name"))` so this can't be null.